src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -81,6 +82,13 @@ public UniquePersonList(UniquePersonList source) {
         return Collections.unmodifiableList(internalList);
     }
 
+    public List<ReadOnlyPerson> sortedListView() {
+        List<ReadOnlyPerson> newInternalList = new ArrayList<>(internalList);
+        Collections.sort(newInternalList, (ReadOnlyPerson p1, ReadOnlyPerson p2) -> p1.getName().fullName.compareToIgnoreCase(p2.getName().fullName));
```

Comment:
Looks good!

Currently only sorted by name, may want to consider adding the ability to sort by other fields like address, etc